test-data.js

```diff
@@ -4,3 +4,21 @@ exports.user = {
   email: email,
   password: 'Password123'
 }
+
+exports.submitPageData = {
+  downloadPDFLink: 'Download the sealed claim form',
+  claimIssuedText: 'Your claim has been issued',
+  emailConfirmation: 'We\'ve emailed confirmation to: ',
+  feesPaid: 'Fee paid: £455',
+  followSteps: 'Follow these steps to serve a claim:',
```

Comment:
I don't think this is a good approach to be verifying this sort of static data on the page.
It ties the implementation too closely to the tests and means that content changes will need test changes for things that shouldn't need it, which makes it harder for content designers to update tests.

Anything calculated makes sense to do here like the fee, but static text more than a heading should not be verified in my opinion, 
@pragnesh-patel  or @hmcts/cmc any other thoughts?

Reply:
I agree static data on page text is prone to making tests brittle.
Also not gd practice in hard-coding values such as fee paid directly in SubmitPageData, It should come from another object in TestData such as export.fees or export.claim

Reply:
If we don't verify the text how do we ensure that the correct text has been displayed on page?

Reply:
the other problem is as we are not inside the main project - we have to write text here- otherwise one content file should take care for tests as well.No need to maintain separate

Reply:
the text is static, the only reason for it to change is if there was a decision to change it.
No need to test static text in general IMO.

having it in the same project wouldn't help as all text goes directly on the page.
Content files are not great practice as it makes it hard to follow what is actually going on the  page

Reply:
I agree with you Tim, however we have to verify the text - if we are checking in unit level - will remove them from here .The one problem is  If we don't verify the text -incase if some one while updating mistakenly updated another key(I am assuming we use Key Value pairs) or completely miss - we are going to see wrong text on page or we are going to miss the content

Reply:
there's no keys to be updated the text is on the page.
any change goes through codereview.